### ISSUE TITLE
glue.create_table with pyarrow schema extracted from csv dataframe

### ISSUE THREAD

**Issue by JPFrancoia** [awslabs/aws-data-wrangler#141]
Hi,

I'm using aws-data-wrangler to load a csv file stored in S3 into a dataframe. Pretty classic. Then I extract the schema of this dataframe like this:

```python
schema_wr = wr.data_types.extract_pyarrow_schema_from_pandas(df, preserve_index=False)
```

This works, the output looks like this:

```python
[(' FOO', DataType(string)), ('BAR', DataType(int64))]
```

The final step is to create a table in Glue with the extracted schema, along with other information:

```python
wr.glue.create_table(
    database="my_database",
    table="test_schema",
    schema=schema_wr,
    path="s3://my_path",
    file_format="csv",
    compression=None,
    partition_cols_schema=[],
   # extra_args={"serde": "LazySimpleSerDe"},
)
```

The table is indeed created in the Glue catalog, but looking at the type of the columns, all of them are `string`:

```
FOO string
BAR string
```

I would rather expect:

```
FOO string
BAR int64
```

First of all, is it the expected behavior?

I think I was able to track down the "issue" to this function: https://github.com/awslabs/aws-data-wrangler/blob/85dba767857eeaec4f01674a9a80b18e6e93f35b/awswrangler/glue.py#L323

And more precisely these two lines:

https://github.com/awslabs/aws-data-wrangler/blob/85dba767857eeaec4f01674a9a80b18e6e93f35b/awswrangler/glue.py#L344

https://github.com/awslabs/aws-data-wrangler/blob/85dba767857eeaec4f01674a9a80b18e6e93f35b/awswrangler/glue.py#L350

We can clearly see that the column type will be "string" all the time with the `OpenCSVSerDe` serde.

I tried with the `LazySimpleSerDe` serde, but then creating the table fails with:

```python
ParamValidationError: Parameter validation failed:
Invalid type for parameter TableInput.StorageDescriptor.Columns[6].Type, value: double, type: <class 'pyarrow.lib.DataType'>, valid types: <class 'str'>
Invalid type for parameter TableInput.StorageDescriptor.Columns[7].Type, value: double, type: <class 'pyarrow.lib.DataType'>, valid types: <class 'str'>
```

Do you have any idea how I can work around that? Happy to test/PR.

Cheers

**Comment by igorborgest** [comment]
Hi @JPFrancoia! Thanks for one more contributions here.

Yeah, unfortunately `OpenCSVSerDe` only accepts strings.
I will address this functionality soon on version `1.0.0` given focus on `LazySimpleSerDe`.

I tell you when it's able to be tested!

**Comment by JPFrancoia** [comment]
Hi @igorborgest , thanks for taking care of the issue!

Yes please let me know, I'll test.

Cheers

**Comment by igorborgest** [comment]
Thanks @JPFrancoia, this issue is resolved on version `1.0.2`. Please, give feedback when possible.

[This](https://github.com/awslabs/aws-data-wrangler/blob/1.0.2/testing/test_awswrangler/test_data_lake.py#L815) test address your use case.

**Comment by JPFrancoia** [comment]
Hi @igorborgest , I saw the releases went up to 1.x a few days ago. I'll start testing! thanks
